src/components/CreateVault.tsx

```diff
@@ -1,44 +1,49 @@
 import { useCallback } from 'react';
 import { useVaultStore, ViewMode, viewModeAtom } from 'store/vaults';
 import { useSetAtom } from 'jotai';
-import CollateralChoice from './CollateralChoice';
+import CollateralChoice from 'components/CollateralChoice';
+import ConfigureNewVault from 'components/ConfigureNewVault';
+import NewVaultOfferSummary from 'components/NewVaultOfferSummary';
 
-const CollateralChoices = () => {
+const CreateVault = () => {
   const setMode = useSetAtom(viewModeAtom);
   const { vaultManagerIds } = useVaultStore();
 
   const buttonProps = {
-    text: 'Back to Manage Vaults',
+    text: 'Cancel',
     onClick: useCallback(() => setMode(ViewMode.Manage), [setMode]),
   };
 
   return (
     <>
-      <div className="flex justify-between mt-6">
+      <div className="flex justify-between mt-6 flex-wrap gap-4">
         <div className="font-serif font-medium text-2xl">
           Creating New Vault
         </div>
         <button
-          className="text-[#f9fafe] text-xs uppercase flex flex-row justify-center items-center p-3 bg-interPurple rounded-md shadow-[0_10px_14px_-4px_rgba(183,135,245,0.3)]"
+          className="transition mr-1 text-[#A3A5B9] uppercase rounded-[6px] border-2 border-solid border-[#A3A5B9] py-2 px-7 leading-[14px] font-semibold text-xs bg-[#A3A5B9] bg-opacity-0 hover:bg-opacity-10 active:bg-opacity-20"
```

Comment:
why are the colors of this button bespoke? is it really different than all the other buttons will be?

I see it's also used in TableRow.

Reply:
For this and all other tailwind specific comments I filed this issue https://github.com/Agoric/dapp-inter/issues/46

I know some of these are reused but it won't be clear to me how many are actually reused, or what an appropriate way to name them is, until I get through all the components. And we don't have all the finalized designs yet even.

I also know some of them are very close to existing classes but right now I don't want to spend time with trial and error seeing if the changes are noticeable enough.

This is worth addressing but I think can be put off, it's a symptom of not being given a design system to work from but rather mockups.